app/models/output_element.rb

```diff
@@ -107,6 +107,18 @@ def has_description?
     description.present?
   end
 
+  # some charts only have one relatee output element, in that case
+  # the hourly/yearly toggle should also show on the current chart
+  def has_one_relatee_output_element?
+    relatee_output_elements.length == 1
```

Comment:
One this occasion, use `count` rather than `length`. `length` will load all of the related output elements and then count them, whereas `count` will run a `SELECT COUNT` query which avoids loading data we don't care about. ActiveRecord also has a [`one?`](https://apidock.com/rails/ActiveRecord/Relation/one%3F) method.

```suggestion
    relatee_output_elements.one?
```